### ISSUE TITLE
Issues in find-variants.sh

### ISSUE THREAD

**Issue by FraBomba6** [biostars/biostar-handbook-issues#261]
## Describe the bug

- [x] On line 46, having the equals between the options and the arguments is not supported by `efetch`.
- [x] On line 77, `samtools mpileup` doesn't support the options `-u` and `-v` (in the [documentation](http://www.htslib.org/doc/samtools-mpileup.html) there's no mention about them). This second issue is hard to auto correct because I don't know what should do the two options. Should the script use `bcftools` just to be consistent with what present in Chapter XXI?

## To Reproduce
Download the script and try to run it following the instructions on the book in Chapter XXI, Section "Variant annotation and effect prediction".

## Desktop
 - OS: macOS
 - ISA-Arch: ARM-64

**Comment by FraBomba6** [comment]
## Update
The same two problems are found also in the script `compare-variant-callers.sh`

**Comment by ialbert** [comment]
I apologize for the problem, it shows the challenges of bioinformatics where code that worked perfectly as little at 3-4 years ago starts failing with no warning. Such is the life of a bioinformatician.

What happened is that efetch stopped accepting = as a parameter specifier, and samtools mpileup was moved to bcftools mpileup

I have corrected the scripts now, but in reality all those scripts should be treated as obsolete, and I need and will rewrite the entire chapter soon.

The code in the chapter is both obsolete in how it solves the problem as well as the implementation of it.

It does not follow the principles laid out in the Art of Bioinformatics Scripting - because when I originally wrote the chapter the principles were not crystallized in my own mind. 

I would recommend all readers to study Biostar Workflows and the techniques described there. The code in that book goes through a more regular testing of all code. 

The main book chapters provide nice narrative, but it is very difficult to keep that code littered in various chapter up to the same standards as code I myself use in my work.
